shiva/src/main/scala/nl/lumc/sasc/biopet/pipelines/shiva/variantcallers/HaplotypeCallerGvcf.scala

```diff
@@ -124,6 +124,7 @@ class HaplotypeCallerGvcf(val parent: Configurable) extends Variantcaller {
     }
 
     genotypeGvcfs.outputDir = outputDir
+    genotypeGvcfs.namePrefix = s"$namePrefix.$name"
```

Comment:
namePrefix is defined as prefix+name ? Wouldn't the variable than better be called genotpeGcvfs.nameWithPrefix?